pages/linux/wpa_passphrase.md

```diff
@@ -0,0 +1,11 @@
+# wpa_passphrase
+
+> Generate a WPA PSK from an ASCII passphrase for a SSID.
+
+- Display WPA PSK for a SSID reading passphrase from stdin:
```

Comment:
```suggestion
- Compute and display the WPA-PSK key for a given SSID reading the passphrase from stdin:
```
I think only saying "display" could be interpreted as "the key is already stored somewhere". Let's say "*compute* and display" instead.